### PR TITLE
CI: enable firefly osx

### PR DESCRIPTION
**Pull request by bsipocz**
This is to test the fix from https://github.com/Caltech-IPAC/firefly_client/pull/61

More context: https://github.com/Caltech-IPAC/irsa-tutorials/pull/10#issuecomment-2263766740 and below

cc @jaladh-singhal

### CODE REVIEW

**Comment by bsipocz** [comment]
This is super low stake (we do run the tests with the linux VM), so I'll just wait for the new release before merging this.

**Comment by bsipocz** [comment]
> On a related thought, can you trigger your pipeline once I have that upcoming PR merged for a final test?

Sure, and I'm also happy to trigger it on your PR before merging it.

**Comment by bsipocz** [comment]
Ahh, this smoked out a bug in the new CI system :)

File: tutorials/requirements.txt [hunk 1]
```diff
@@ -11,9 +11,7 @@ dask[distributed]
 psutil
 ray
 s3fs
-firefly-client
+git+https://github.com/Caltech-IPAC/firefly_client@FIREFLY-1527-update-package-config#egg=firefly_client
```
Comment (jaladh-singhal) [inline tutorials/requirements.txt:14]: This can now safely go back to firefly-client. Thanks for all the testing!

**Comment by jaladh-singhal** [comment]
Great! all checks passed

**Pull request merged**